http-gateways/PATH_GATEWAY.md

```diff
@@ -563,6 +564,13 @@ Optional, present in certain response types:
   non-executable binary response types are not used in `<script>` and `<style>`
   HTML tags.
 
+### `X-Trace-Id` (response header)
+
+Optional. Implementations are free to use this header to return a globally
+unique identifier to help in debugging errors and performance issues.
+
+A good practice is to always return it with HTTP error [status codes](#response-status-codes) >=`400`.
+
```

Comment:
@Arqu  fyi added this to https://github.com/ipfs/specs/pull/283  – having a suggestion of 'best practice' in official specs will simplify debugging across ecosystem